rest_framework/exceptions.py

```diff
@@ -58,6 +59,14 @@ def __str__(self):
         return self.detail
 
 
+def build_error_from_django_validation_error(exc_info):
+    code = exc_info.code or 'invalid'
```

Comment:
I believe this should be:

`code = getattr(exc_info, 'code', None) or 'invalid'`